tests/unit/customizations/test_cloudtrail.py

```diff
@@ -46,6 +46,14 @@ def test_create_subscription_has_zero_rc(self):
         # sure it says log delivery is happening.
         self.assertIn('Logs will be delivered to foo', stdout)
 
+    def test_policy_from_paramfile(self):
+        with temporary_file('w') as f:
+            f.write('{"Statement": []}')
+            command = (
+                'cloudtrail create-subscription --s3-use-bucket foo '
+                '--name bar --s3-custom-policy file://{0}'.format(f.name))
```

Comment:
I agree with Kyle here.  Let's verify that the policy we pass into the botocore layer is the value we expect.